### PR TITLE
Update Database

### PR DESCRIPTION
**Pull request by fabianofrank**
Simulated the operations of creating a database and added data to it for a PostgreSQL knowledge check.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,11 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
+SELECT name FROM animals WHERE date_of_birth BETWEEN 2016 AND 2019;
+SELECT name FROM animals WHERE neutered < 3;
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10;
+SELECT * FROM animals WHERE neutered = true;
+SELECT * FROM animals WHERE name NOT IN ('Gabumon');
```
Comment (AnselemOdims) [inline queries.sql:9]: - [ ] This is actually right and can also be done with

```sql
SELECT * FROM animals WHERE name <> 'Gabumon';
```

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+SELECT * FROM animals WHERE name != 'Gabumon';
```
Comment (AnselemOdims) [inline queries.sql:10]: - [ ] [OPTIONAL] Please remove this as the other one should do the job

**Review by AnselemOdims — APPROVED** [review]
### Hi  @fabianofrank 🙋‍♀️, 
Good job implementing all the project requirements for this milestone/project.

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 2 above)
Comment (fabianofrank) [reply queries.sql:10]: thanks man

**Pull request merged**